### PR TITLE
Fixed readme syntax highlighting, updated examples to es6

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

**Comment by jacwright** [comment]
I appreciate the PR, but I think the examples should stay old school. ES6 provides promises natively so this may not an important library for those using ES6.

I would accept the syntax highlighting part.

If you're interested in contributing to this library more, the API needs to be updated to come inline with ES6 Promises (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). The old API should be deprecated, not removed entirely, to avoid breaking modules that rely on promises.

**Comment by amilajack** [comment]
Staying with ES5 is fine. Should I migrate the examples back to ES5 but keep the syntax highlighting?

**Comment by jacwright** [comment]
Sure!

File: readme.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ function ifServiceOffLineUseCache(err) {
     }
 }
 
-db.load().rejected(ifOfflineUseCache).then(function(data) {
+db.load().rejected(ifOfflineUseCache).then(function(data) => {
```
Comment (jacwright) [inline readme.md]: Missed a spot when removing the ES6.

**Comment by amilajack** [comment]
Fixed!

**Comment by jacwright** [comment]
Thank you for the contribution!

**Pull request merged**